### PR TITLE
Simulare doc

### PR DESCRIPTION
**Pull request by shivani61**
- Added simulare intro on dashboard and devices

### CODE REVIEW

File: docs/home/devconsole/device-management/index.md [hunk 1]
```diff
@@ -16,6 +16,22 @@ There are checkboxes on the top of the page with categories such as Active, Inac
 
 Search is provided on the top right where you can search for devices in Devices using the Esper assigned device code which is displayed on the device's home page wallpaper.
 
+## Getting Started
+
+If you are logging in to your endpoint for the very first time you will see the following screen which gives you a choice to either provision a device or Simulate a device.
+
+![Simulare](../images/simulare-1.png)
+
+You can choose to provision a device the details to which are provided in the Provision Device section of this documentation. Or you may choose to Simulate a Device. You will see a momentary "You are all set!" message post which a mock device will be available for you to experiment and play around with till you provision an actual device.
```
Comment (bmehendale) [inline docs/home/devconsole/device-management/index.md]: `post which` --> `, after which`

File: docs/home/devconsole/device-management/index.md [hunk 2]
```diff
@@ -16,6 +16,22 @@ There are checkboxes on the top of the page with categories such as Active, Inac
...
+
+![Simulare](../images/simulare-2.png)
+
+Simulare helps you explore the Esper platform without using a real device. We provide this simulated device to act as an actual device to help you experience the capabilities of our platform. 
+
+Capabilities such as Apps, Remote View and Capture log might not be available on the Simulated device.
```
Comment (bmehendale) [inline docs/home/devconsole/device-management/index.md]: `the Simulated device.` -> `a Simulated device.`

File: docs/home/devconsole/device-management/index.md [hunk 3]
```diff
@@ -16,6 +16,22 @@ There are checkboxes on the top of the page with categories such as Active, Inac
...
+
+You can choose to provision a device the details to which are provided in the Provision Device section of this documentation. Or you may choose to Simulate a Device. You will see a momentary "You are all set!" message post which a mock device will be available for you to experiment and play around with till you provision an actual device.
+
+![Simulare](../images/simulare-2.png)
+
+Simulare helps you explore the Esper platform without using a real device. We provide this simulated device to act as an actual device to help you experience the capabilities of our platform. 
+
+Capabilities such as Apps, Remote View and Capture log might not be available on the Simulated device. 
+
+This device is only available for new signups currently.
```
Comment (bmehendale) [inline docs/home/devconsole/device-management/index.md]: `This device is only available for new signups currently.` --> `Please note that trial accounts created on or after September 27, 2019 will have this feature enabled. Accounts created before that will not. Please reach out to us at support@esper.io if you signed up before September 27, and would like to try out Simulated devices.`

File: docs/home/devconsole/device-management/index.md (hunk 1 above)
Comment (bmehendale) [inline docs/home/devconsole/device-management/index.md]: ` choose to provision a device` --> `choose to provision a real Android device` 

`in the Provision Device section of this documentation` --> have url to section for provisioning devices inline with text

Comment (shivani61) [reply docs/home/devconsole/device-management/index.md]: changed

File: docs/home/devconsole/device-management/index.md (hunk 2 above)
Comment (shivani61) [reply docs/home/devconsole/device-management/index.md]: changed

File: docs/home/devconsole/device-management/index.md (hunk 3 above)
Comment (shivani61) [reply docs/home/devconsole/device-management/index.md]: added

File: docs/home/devconsole/device-management/index.md (hunk 1 above)
Comment (shivani61) [reply docs/home/devconsole/device-management/index.md]: added

**Pull request merged**